docs/technical-setup/install-a-git-client.md

```diff
@@ -51,17 +101,16 @@ For this, launch a Terminal window, enter the command shown below, and follow th
 xcode-select --install
 ```
 
-<!--
-### Install Git on Windows
+### Install Git on Windows in WSL2
 
-We recommend installing the [Windows Subsystem for Linux (WSL)](https://learn.microsoft.com/en-us/windows/wsl/install) with the default Ubuntu Linux distribution.
-You can then install Git into your Ubuntu system:
+Once you have [installed WSL2](./install-wsl2.md), you can install Git using `apt`:
+The [WSL2 installation itself](./install-wsl2.md) comes with Git, so no further action is needed.
 
-```
-# You may need to update the package list first:
-# apt-get update
+That said, if you want to update the Git installation, please refer to [Git's documentation directly](https://git-scm.com/download/linux).
```

Comment:
If Git is really installed by default, I would take this section out.

Reply:
> If Git is really installed by default, I would take this section out.

Oh this was my bad. Turns out it's not installed by default and I have some old sentence like this here that should be removed.

Reply:
Are you sure? All the docs I can find say it should be included.

Reply:
I know, right? They do! But when I first went to run `git` it wasn't there, which is why I installed it with `apt`. It's quite challenging it turns out to uninstall WSL and start over to figure this out for sure though... 

Maybe this is the answer: Once the docs are ll set, I'll wipe the computer and follow my instructions from scratch to make sure it's all correct, and as part of that I'll double check git.